src/gt4sd/algorithms/generation/tests/test_diffusion.py

```diff
@@ -242,3 +254,27 @@ def test_generation_via_registry(
     assert len(samples) == 1
     sample = samples[0]
     assert isinstance(sample, PIL.Image.Image)
+
+
+@pytest.mark.parametrize(
+    "config, algorithm",
+    [
+        pytest.param(
+            GeoDiffGenerator,
+            DiffusersGenerationAlgorithm,
+        ),
+    ],
+)
+def test_geodiff_conditional_generation_via_import(config, algorithm):
+    with open(
+        "/home/runner/work/gt4sd-core/gt4sd-core/src/gt4sd/algorithms/generation/tests/mol_dct.pkl",
```

Comment:
use the pattern used in all the other tests by adding the file to `setup.cfg` and reading it from the package. In this way we are CI/filesystem agnostic.